command/genesis/params.go

```diff
@@ -187,6 +191,8 @@ func (p *genesisParams) validateFlags() error {
 		}
 	}
 
+	params.stakeTokenAddr = types.StringToAddress(params.stakeToken)
```

Comment:
Add validation of `stakeToken` address, like we have in other commands. (Use `types.IsValidAddress(contractAddress)` function).